azure-quantum/azure/quantum/qiskit/job.py

```diff
@@ -326,10 +346,14 @@ def _format_microsoft_v2_results(self) -> List[Dict[str, Any]]:
         entry_point_names = self._get_entry_point_names()
 
         results = self._translate_microsoft_v2_results()
-        
+
+        headers = self._get_headers()

```

Comment:
nit: since the `headers` variable is not used until after the first `if` statement, its assignment can be moved until after it.